packages/blockchain-wallet-v4-frontend/src/modals/EarnCompare/index.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import {
+  Button,
+  Flex,
+  IconCloseCircleV2,
+  Padding,
+  SemanticColors,
+  Text
+} from '@blockchain-com/constellation'
+
+import { useRemote } from 'hooks'
+import modalEnhancer from 'providers/ModalEnhancer'
+
+import Loading from './EarnCompare.loading.template'
+import { tableData } from './EarnCompare.model'
+import { getRemote } from './EarnCompare.selectors'
+import { CustomModal, TableContainer } from './EarnCompare.styles'
+import TableRow from './TableRow'
+import { TableRowPropsType } from './TableRow/TableRow.types'
+
+const EarnCompare = ({ closeAll }: OwnProps) => {
+  const { data } = useRemote(getRemote)
+  if (!data) return <Loading />
```

Comment:
Do we want to display loading in case of an error too?

Reply:
Good call. We want to make the values '-' if data is error or loading

Reply:
It does not feel very helpful from UX perspective in case of an error, but I assume that is what product wanted.